library/src/main/java/com/devbrackets/android/exomedia/core/api/VideoViewApi.java

```diff
@@ -112,6 +112,10 @@
 
     void setTrack(ExoMedia.RendererType type, int trackIndex);
 
+    void setRendererTrackGroupIndex(ExoMedia.RendererType trackType, int trackGroupIndex);
```

Comment:
This looks like it's a duplicate of `setTrack`. Why did you need to define a separate function for this?